### ISSUE TITLE
some questions

### ISSUE THREAD

**Issue by LREM-C** [mellody11/PCAM-With-RandLA-Net#1]
Why is num_workers not set in the dataloader? If the default is 0, it seems that the memory utilization rate has always been low and maintained at around 20.

**Comment by liuxuexun** [comment]
Thank you for pointing out this issue. As you said, do not set num_ Workers will lead to low memory and GPU usage.

This repository was my previous work and I just uploaded it recently. I didn't realize this problem at that time. Thank you for pointing out that.

The code has been updated.

**Comment by LREM-C** [comment]
But when I set it up, the final performance will be greatly affected. I don’t know what your test results are. Can you verify it?

**Comment by liuxuexun** [comment]
My test results are base the code without num_workers, I will release those soon.

During my study resently, I also meet this problem in another work. But I'm not sure if it is due to the influence of num_workers.

It seems that the selecting points method based on **probability** can not use the worker? However I am not sure.

if you use SemanticKITTI datasets, it does not have this problem, because the selection of training and validation data is not based on **probability**

**Comment by LREM-C** [comment]
Yes, this problem has troubled me for a long time, if so, how to set up to improve utilization

**Comment by liuxuexun** [comment]
I am sorry to say I have not any idea yet.

Maybe you can find the answer from the recent published work.
